pkg/cache/sql/informer/listoption_indexer.go

```diff
@@ -246,15 +263,15 @@ func (l *ListOptionIndexer) ListByOptions(ctx context.Context, lo ListOptions, p
 	}
 
 	if len(whereClauses) > 0 {
-		stmt += "\n  WHERE\n    "
+		query += "\n  WHERE\n    "
 		for index, clause := range whereClauses {
-			stmt += fmt.Sprintf("(%s)", clause)
+			query += fmt.Sprintf("(%s)", clause)
 			if index == len(whereClauses)-1 {
 				break
 			}
-			stmt += " AND\n    "
+			query += " AND\n    "
 		}
-		// stmt += strings.Join(whereClauses, " AND\n    ")
+		// query += strings.Join(whereClauses, " AND\n    ")
```

Comment:
Do we need this line to remain commented out?

Reply:
Addressed in current master